doc/release-notes/official.md

```diff
@@ -795,7 +795,7 @@
  - 2018-07-27 - [2 commits](https://github.com/mockito/mockito/compare/v2.20.1...v2.20.2) by [Rafael Winterhalter](https://github.com/raphw) (1), [Szczepan Faber](https://github.com/mockitoguy) (1) - published to [![Bintray](https://img.shields.io/badge/Bintray-2.20.2-green.svg)](https://bintray.com/mockito/maven/mockito-development/2.20.2)
  - Updates Byte Buddy to add preliminary support for Java 12. [(#1448)](https://github.com/mockito/mockito/pull/1448)
 
-# 2.20.1 (2018-07-26) published to [JCenter](https://bintray.com/mockito/maven/mockito/2.20.1)/[Maven Central](http://search.maven.org/#artifactdetails%7Corg.mockito%7Cmockito-core%7C2.20.1%7Cjar)
+# 2.20.1 (2018-07-26) published to [JCenter](https://bintray.com/mockito/maven/mockito/2.20.1)/[Maven Central](https://search.maven.org/artifact/org.mockito/mockito-core/2.20.1/jar)
```

Comment:
Let's not update our release notes retroactively. Could you please revert the changes in this file?

Reply:
Reverted it in the latest commit. I have used a [fixup](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---fixupltcommitgt) commit for that, though I guess that was unnecessary since you will squash the commits anyways.

Should I also add `[skip release]` to the commit message since there are nearly no code changes?